### PR TITLE
feat: implement a `wait` action

### PR DESCRIPTION
**Pull request by jpaulo789b**
Addition of a wait option, some apps that take a few seconds to enter the splash may not work tests without this action

## Status

**READY**

## Description

Add new action label wait
`- wait: 1 second` // wait 1 second to make next action
 using plain english durations
 exemples:
 ```
wait: 1 microsecond
wait: 1 millisecond
wait: 1 second
wait: 1 minute
wait: 1 hour
```

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/fluttium_runner/test/src/fluttium_runner_test.dart [hunk 1]
```diff
@@ -84,6 +85,7 @@ description: test
 - expectNotVisible: "Text"
 - inputText: "Text"
 - takeScreenshot: "Text"
+- wait: "Text"
```
Comment (wolfenrain) [inline packages/fluttium_runner/test/src/fluttium_runner_test.dart:88]: This would fail according to the code as it should be an parseable int

Comment (jpaulo789b) [reply packages/fluttium_runner/test/src/fluttium_runner_test.dart:88]: modified to using plain english durations

**Comment by jpaulo789b** [comment]
modified to using plain english durations

**Comment by jpaulo789b** [comment]
Do you need me to modify anything else?

**Comment by wolfenrain** [comment]
Hi @jpaulo789b, I finally got around to merging #112 which is a full rewrite of Fluttium's action system.

If you want you can recreate your PR using the new action API, or I can add one to the core system next week. Sorry for the inconvenience, your contribution is still appreciated.

**Pull request closed without merging**